dir-index-uncat.html

```diff
@@ -2,6 +2,8 @@
 <html>
 <head>
   <meta charset="utf-8" />
+  <meta name="description" content="Index of {{ .Path }}">
+  <meta name="author" content="Ipfs">
```

Comment:
That's not really what the author tag is for. We don't want to claim that all content stored on IPFS is authored *by* IPFS.

Reply:
It is directory listing, but yeah, I am not sure.

Reply:
It's still "authored" by someone. Someone created the directory and that someone isn't the IPFS daemon; the IPFS daemon just formatted the list for display in the browser.

Reply:
well the author could be the public key of the gateway instance if such a variable its available

Reply:
That's also not really the "author". The author is the person who put together the directory listing. The node/IPFS is just a compiler or translator.